app/src/main/java/org/schabi/newpipe/about/AboutActivity.kt

```diff
@@ -162,6 +162,10 @@ class AboutActivity : AppCompatActivity() {
                 "OkHttp", "2019", "Square, Inc.",
                 "https://square.github.io/okhttp/", StandardLicenses.APACHE2
             ),
+            SoftwareComponent(
+                "Picasso", "2013", "Square, Inc.",
```

Comment:
It should be 2013-2020

Reply:
According to their [README](https://github.com/square/picasso#license): "Copyright 2013 Square, Inc."

I'd just remove all `- 2020`, since they are already outdated and it doesn't make sense that we update them on each new year